Moblin/HaishinKit/Media/AudioUnit.swift

```diff
@@ -175,6 +169,35 @@ final class AudioUnit: NSObject {
     func removeReplaceAudioInner(cameraId: UUID) {
         replaceAudios.removeValue(forKey: cameraId)
     }
+
+    func prepareSampleBuffer(sampleBuffer: CMSampleBuffer, audioLevel: Float, numberOfAudioChannels: Int) {
+        // Workaround for audio drift on iPhone 15 Pro Max running iOS 17. Probably issue on more models.
+        let presentationTimeStamp = syncTimeToVideo(mixer: mixer!, sampleBuffer: sampleBuffer)
```

Comment:
`mixer!` means the app will crash if `mixer` is `nil`. I mean, it can probably never be `nil` here, but I think it's a good habit to minimize the use of `!` to reduce the risk of app crashes.

The big exception to this is for settings (the database). There everything is expected to exist, at least in the vast majority of cases.